### PR TITLE
Add tools variant in caliper

### PR DESCRIPTION
**Pull request by victorapm**
This pull request includes updates to the `caliper` package to add a new variant for enabling tools support. Note that tools support is currently enabled by default in caliper's `CMakeLists`

### CODE REVIEW

**Review by alalazo — APPROVED** [review]

**Comment by alalazo** [comment]
Waiting for maintainers approval before merging

**Review by daboehme — APPROVED** [review]
Looks good to me

**Pull request merged**
